client/extensions/auto-translator/src/widget/translation-dialog/compare-accordion/index.tsx

```diff
@@ -0,0 +1,226 @@
+import DiffMatchPatch from "diff-match-patch";
+import { useFormikContext } from "formik";
+import * as React from "react";
+import { Container, GridList, ToggleBox } from "superdesk-ui-framework/react";
+import { Select } from "../../../components";
+import { superdesk } from "../../../superdesk";
+import { capitalize, getObjectKeys } from "../../../utilities";
+import { TranslationDialogFormProps } from "../helpers";
+
+const COMPARE_VERIONS = ["left", "right", "diff"] as const;
+
+const CompareContent = ({
+  headline,
+  headline_extended,
+  body_html,
+  index,
+}: Pick<
+  TranslationDialogFormProps["translations"][string]["original"],
+  "headline" | "headline_extended" | "body_html"
+> & { index: number }) => {
+  const { gettext } = superdesk.localization;
+
+  return (
+    <>
+      <div className="sd-input sd-input--medium sd-input--boxed-style sd-input--boxed-label">
+        <span
+          className="sd-input__label sd-input__label--boxed"
+          id={`compare-headline-${index}-label`}
+        >
+          {capitalize(gettext("headline"))}
+        </span>
+        <div className="sd-input__input-container">
+          <p
+            aria-labelledby={`compare-headline-${index}-label`}
+            className="m-0"
+            dangerouslySetInnerHTML={{ __html: headline }}
```

Comment:
Should we be concerned about cross scripting? If so should we use a library to sanitize inputs? I found this one https://www.npmjs.com/package/dompurify

Reply:
Nevermind, you implemented already in the last commit